src/GNNGraphs/transform.jl

```diff
@@ -40,18 +40,19 @@ end
 
 """
     add_self_loops(g::GNNHeteroGraph, edge_t::EType)
+    add_self_loops(g::GNNHeteroGraph)
 
-If the source node type is the same as the destination node type in `edge_t`,
-return a graph with the same features as `g` but also add self-loops 
-of the specified type, `edge_t`. Otherwise, it returns `g` unchanged.
-
-Nodes with already existing self-loops of type `edge_t` will obtain 
-a second set of self-loops of the same type.
-
-If the graph has edge weights for edges of type `edge_t`, the new edges will have weight 1.
+- For a specific edge type (`edge_t`):
```

Comment:
Let's not start a docstring with a bullet point. 
The change to the docstring can be minimal, just adding a paragraph at the end saying "If `edge_t` is not passed as argument, ..."

Reply:
Done